dev_guide/builds.adoc

```diff
@@ -282,26 +287,26 @@ For example disabling assets compilation for your Rails application:
 ====
 
 == Build Triggers
-When defining a `*BuildConfig*`, you can define triggers to control the
-circumstances in which a build should be run for the `*BuildConfig*`. There are two
-types of triggers available:
+When defining a `*buildConfig*`, you can define triggers to control the
+circumstances in which the `*buildConfig*` should be run. There are two types of
+triggers available:
 
 * link:#webhook-triggers[Webhook]
 * link:#image-change-triggers[Image change]
 
 === Webhook Triggers
 Webhook triggers allow you to trigger a new build by sending a request to the
-OpenShift API endpoint. You can define these triggers using
+OpenShift API endpoint. Define these triggers using
```

Comment:
Possibly leave this as "You can define" since it's optional. But maybe it's fine this way too.